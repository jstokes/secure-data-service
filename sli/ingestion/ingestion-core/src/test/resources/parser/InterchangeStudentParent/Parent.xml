<?xml version="1.0" encoding="UTF-8"?>
<InterchangeStudentParent xmlns:sli="http://slc-sli/ed-org/0.1" xmlns="http://ed-fi.org/0100"
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
    <Parent id="ID003">
        <ParentUniqueStateId>L0CRWfT6iKxuBlI-_3DHMRQ</ParentUniqueStateId>
        <Name Verification="Life insurance policy">
            <PersonalTitlePrefix>Sr</PersonalTitlePrefix>
            <FirstName>qPN1QmqDdNt2oGlEkJL</FirstName>
            <MiddleName>G5rY1qTTtbadAYw3G0lrgpHuAUY-lvBMu_A6DbNkJSGCZZ6VDzK.xvRTfvO</MiddleName>
            <LastSurname>NNTCJstdqruOQ6K9</LastSurname>
            <GenerationCodeSuffix>VI</GenerationCodeSuffix>
            <MaidenName>FqyUP4fSrqqny4Ay_G8XaYYbwWCnb_gY</MaidenName>
        </Name>
        <OtherName OtherNameType="Alias">
            <PersonalTitlePrefix>Mrs</PersonalTitlePrefix>
            <FirstName>SOEFxP1-8AVUjFr3b_d_RVwsK</FirstName>
            <MiddleName>hsDvxcxC-KeruW-zaHns3oaG2FwxcQQjmQX87K8TPChubfs0I96rx5uf.IeEr</MiddleName>
            <LastSurname>vR0YGhFyws2fWviMzo-</LastSurname>
            <GenerationCodeSuffix>Sr</GenerationCodeSuffix>
        </OtherName>
        <OtherName OtherNameType="Alias">
            <PersonalTitlePrefix>Sister</PersonalTitlePrefix>
            <FirstName>J-eU4orLu.5d_w-tBJKOfmqbVZYUxXTgvJp_ursTUQr5u74ypJ76zEb-xDw</FirstName>
            <MiddleName>Qh4eIcw-PkYgO0N3FSKLxgrUfC3qluAcv2p8FD4aOWYs</MiddleName>
            <LastSurname>njtct7g3pC5d</LastSurname>
            <GenerationCodeSuffix>VII</GenerationCodeSuffix>
        </OtherName>
        <Sex>Female</Sex>
        <Address AddressType="Mailing">
            <AddressLine>TdCVN8RLefbvfv8K-xAntOId3M0sASnso5nwnlk-M3NUOH6w4nUDlp4IcqIWLLjBSwp7o</AddressLine>
            <AddressLine>VBmxxBB9v_ZHe3NrWaXRyqx2gUE</AddressLine>
            <Country>CANADA</Country>
            <Latitude>deB6kPdObUK-kM_</Latitude>
            <Longitude>GWe</Longitude>
            <BeginDate>2014-01-25</BeginDate>
            <EndDate>2014-07-09</EndDate>
        </Address>
        <Address AddressType="Billing">
            <StreetNumberName>r</StreetNumberName>
            <ApartmentRoomSuiteNumber>geAcoxDH2IoL9WFheP7d</ApartmentRoomSuiteNumber>
            <BuildingSiteNumber>xNrLneiPUy4wTs.mOBO.</BuildingSiteNumber>
            <City>jBsbclSdC5kXqGHeW</City>
            <StateAbbreviation>UT</StateAbbreviation>
            <PostalCode>IbqD</PostalCode>
            <NameOfCounty>OQdr-Rzbo9PGXUubI60rMVHR</NameOfCounty>
            <CountyFIPSCode>WLwKp</CountyFIPSCode>
            <Latitude>J</Latitude>
            <Longitude>A9K3PUcHCSG7N</Longitude>
            <BeginDate>2013-12-29</BeginDate>
            <EndDate>2013-04-06</EndDate>
        </Address>
        <Telephone TelephoneNumberType="Unlisted" PrimaryTelephoneNumberIndicator="0">
            <TelephoneNumber>vq4</TelephoneNumber>
        </Telephone>
        <Telephone TelephoneNumberType="Emergency 1" PrimaryTelephoneNumberIndicator="0">
            <TelephoneNumber>gPvWtP6k6VBSVJqUh</TelephoneNumber>
        </Telephone>
        <ElectronicMail EmailAddressType="Work">
            <EmailAddress>vj9ZXsMdvOCTifTMe36G5cV9fYmVp9x</EmailAddress>
        </ElectronicMail>
        <ElectronicMail EmailAddressType="Other">
            <EmailAddress>uREWY5bBJLSnbgbCzRnwVHSGSg</EmailAddress>
        </ElectronicMail>
        <LoginId>SshLa9d5WPUqDQnoUks4kd.HFuTtODf0BGWxrdz5</LoginId>
    </Parent>
</InterchangeStudentParent>
