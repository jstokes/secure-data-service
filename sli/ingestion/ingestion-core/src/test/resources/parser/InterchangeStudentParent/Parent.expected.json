{
  "@id":"ID003",
  "ParentUniqueStateId":{
    "_value":"L0CRWfT6iKxuBlI-_3DHMRQ"
  },
  "Name":{
    "@Verification":"Life insurance policy",
    "PersonalTitlePrefix":{
      "_value":"Sr"
    },
    "FirstName":{
      "_value":"qPN1QmqDdNt2oGlEkJL"
    },
    "MiddleName":{
      "_value":"G5rY1qTTtbadAYw3G0lrgpHuAUY-lvBMu_A6DbNkJSGCZZ6VDzK.xvRTfvO"
    },
    "LastSurname":{
      "_value":"NNTCJstdqruOQ6K9"
    },
    "GenerationCodeSuffix":{
      "_value":"VI"
    },
    "MaidenName":{
      "_value":"FqyUP4fSrqqny4Ay_G8XaYYbwWCnb_gY"
    }
  },
  "OtherName":[
    {
      "@OtherNameType":"Alias",
      "PersonalTitlePrefix":{
        "_value":"Mrs"
      },
      "FirstName":{
        "_value":"SOEFxP1-8AVUjFr3b_d_RVwsK"
      },
      "MiddleName":{
        "_value":"hsDvxcxC-KeruW-zaHns3oaG2FwxcQQjmQX87K8TPChubfs0I96rx5uf.IeEr"
      },
      "LastSurname":{
        "_value":"vR0YGhFyws2fWviMzo-"
      },
      "GenerationCodeSuffix":{
        "_value":"Sr"
      }
    },
    {
      "@OtherNameType":"Alias",
      "PersonalTitlePrefix":{
        "_value":"Sister"
      },
      "FirstName":{
        "_value":"J-eU4orLu.5d_w-tBJKOfmqbVZYUxXTgvJp_ursTUQr5u74ypJ76zEb-xDw"
      },
      "MiddleName":{
        "_value":"Qh4eIcw-PkYgO0N3FSKLxgrUfC3qluAcv2p8FD4aOWYs"
      },
      "LastSurname":{
        "_value":"njtct7g3pC5d"
      },
      "GenerationCodeSuffix":{
        "_value":"VII"
      }
    }
  ],
  "Sex":{
    "_value":"Female"
  },
  "Address":[
    {
      "@AddressType":"Mailing",
      "AddressLine":[
        {
          "_value":"TdCVN8RLefbvfv8K-xAntOId3M0sASnso5nwnlk-M3NUOH6w4nUDlp4IcqIWLLjBSwp7o"
        },
        {
          "_value":"VBmxxBB9v_ZHe3NrWaXRyqx2gUE"
        }
      ],
      "Country":{
        "_value":"CANADA"
      },
      "Latitude":{
        "_value":"deB6kPdObUK-kM_"
      },
      "Longitude":{
        "_value":"GWe"
      },
      "BeginDate":{
        "_value":"2014-01-25"
      },
      "EndDate":{
        "_value":"2014-07-09"
      }
    },
    {
      "@AddressType":"Billing",
      "StreetNumberName":{
        "_value":"r"
      },
      "ApartmentRoomSuiteNumber":{
        "_value":"geAcoxDH2IoL9WFheP7d"
      },
      "BuildingSiteNumber":{
        "_value":"xNrLneiPUy4wTs.mOBO."
      },
      "City":{
        "_value":"jBsbclSdC5kXqGHeW"
      },
      "StateAbbreviation":{
        "_value":"UT"
      },
      "PostalCode":{
        "_value":"IbqD"
      },
      "NameOfCounty":{
        "_value":"OQdr-Rzbo9PGXUubI60rMVHR"
      },
      "CountyFIPSCode":{
        "_value":"WLwKp"
      },
      "Latitude":{
        "_value":"J"
      },
      "Longitude":{
        "_value":"A9K3PUcHCSG7N"
      },
      "BeginDate":{
        "_value":"2013-12-29"
      },
      "EndDate":{
        "_value":"2013-04-06"
      }
    }
  ],
  "Telephone":[
    {
      "@PrimaryTelephoneNumberIndicator":"0",
      "@TelephoneNumberType":"Unlisted",
      "TelephoneNumber":{
        "_value":"vq4"
      }
    },
    {
      "@PrimaryTelephoneNumberIndicator":"0",
      "@TelephoneNumberType":"Emergency 1",
      "TelephoneNumber":{
        "_value":"gPvWtP6k6VBSVJqUh"
      }
    }
  ],
  "ElectronicMail":[
    {
      "@EmailAddressType":"Work",
      "EmailAddress":{
        "_value":"vj9ZXsMdvOCTifTMe36G5cV9fYmVp9x"
      }
    },
    {
      "@EmailAddressType":"Other",
      "EmailAddress":{
        "_value":"uREWY5bBJLSnbgbCzRnwVHSGSg"
      }
    }
  ],
  "LoginId":{
    "_value":"SshLa9d5WPUqDQnoUks4kd.HFuTtODf0BGWxrdz5"
  }
}
